tests/dag_processing/test_manager.py

```diff
@@ -570,7 +618,7 @@ def fake_processor_(*args, **kwargs):
             manager = DagFileProcessorManager(
                 dag_directory=test_dag_path,
                 max_runs=1,
-                processor_timeout=timedelta.max,
+                processor_timeout=timedelta(hours=10),
```

Comment:
Setting this value to `max` caused issues due to the following line of code, which led to an overflow:
```python
                    and (dag.last_parsed_time + self._processor_timeout) < last_parsed[dag.fileloc]
```